From 741d77e42fc4af49804f7ee43b7237e01633cbcd Mon Sep 17 00:00:00 2001
From: Roy Li <rongqing.li@windriver.com>
Date: Tue, 8 Jul 2014 13:20:47 +0800
Subject: [PATCH] unnecessary to check libpcap

Upstream-Status: Pending

since the check of libpcap did not consider the cross-compile, lead to the
below error:
	This autoconf log indicates errors, it looked at host include and/or
	library paths while determining system capabilities.

In fact, the libpcap has been added into the tcpdump's DEPENDS, not need to
check if libpcap existed.

Signed-off-by: Roy Li <rongqing.li@windriver.com>

---
 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index b2305a5..b3b5dbf 100644
--- a/configure.in
+++ b/configure.in
@@ -418,7 +418,7 @@ dnl Some platforms may need -lnsl for getrpcbynumber.
 AC_SEARCH_LIBS(getrpcbynumber, nsl,
     AC_DEFINE(HAVE_GETRPCBYNUMBER, 1, [define if you have getrpcbynumber()]))
 
-AC_LBL_LIBPCAP(V_PCAPDEP, V_INCLS)
+#AC_LBL_LIBPCAP(V_PCAPDEP, V_INCLS)
 
 #
 # Check for these after AC_LBL_LIBPCAP, so we link with the appropriate
